packages/terra-date-input/src/DateInputUtil.js

```diff
@@ -76,19 +76,18 @@ class DateInputUtil {
    */
   static incrementYear(year) {
     // Handle empty input value
-    if (year === '') {
-      return '1';
+    let numericYear = Number(year);
+    if (numericYear === 0) {
```

Comment:
What is the reason for allowing to increment to 1900 from year `0` but not year `1`?

Reply:
This condition is to increment when the field is empty.

Reply:
If so, shouldn't this check for an empty string like what it did before instead of checking for 0?

Reply:
I had checked for empty string but value is '0' as I found it after putting a breakpoint. It is because of this [line](https://github.com/cerner/terra-framework/blob/1ea109bb8eabf4a2d92677cd8348213dafe78743/packages/terra-date-input/src/DateInput.jsx#L308)

Reply:
Ah, I see. Should that line in [handleDayKeyDown](https://github.com/cerner/terra-framework/blob/0cbfdf5745af49f9e3b4273a34253078e1dd9aed/packages/terra-date-input/src/DateInput.jsx#L308) and [handleDayKeyDown](https://github.com/cerner/terra-framework/blob/0cbfdf5745af49f9e3b4273a34253078e1dd9aed/packages/terra-date-input/src/DateInput.jsx#L272) be updated to default to `''` instead of `'0'` so that it can't increment from 0. It just seem weird that it currently can increment from 0 but not from 1.

Reply:
Done 3052cfe